creator-kit-all-cascoda.config

```diff
@@ -0,0 +1,14 @@
+CONFIG_TARGET_pistachio=y
+CONFIG_TARGET_pistachio_marduk=y
+CONFIG_TARGET_pistachio_marduk_marduk_ca8210=y
+CONFIG_PACKAGE_button-gateway=y
+CONFIG_PACKAGE_device-manager=y
+CONFIG_PACKAGE_webscripts=y
+CONFIG_ALL=y
```

Comment:
Please remove this as we don't need any project to be enabled by default for build-all configuration